lifetimes/estimation.py

```diff
@@ -278,6 +298,7 @@ def fit(self, frequency, monetary_value, iterative_fitting=4, initial_params=Non
                  This model is not very stable so we suggest >10 for best estimates evaluation.
```

Comment:
bah, good question - thanks for calling me out. I think it was for perf reasons initially